android-components/components/support/base/src/test/java/mozilla/components/support/base/log/sink/AndroidLogSinkTest.kt

```diff
@@ -175,38 +175,6 @@ class AndroidLogSinkTest {
         )
     }
 
-    @Test
-    fun `Sink will print stacktrace of throwable`() {
```

Comment:
This is more of a nit, but since there was a test only for throwable even if it was nullable, now that the message is non-nullable, I was thinking an additional test here would not hurt.

```
    @Test
    fun `Sink will print message`() {
        val sink = AndroidLogSink()

        sink.log(message = "An error occurred")

        val logs = ShadowLog.getLogs()
        assertEquals(1, logs.size)
        assertEquals(
            "An error occurred",
            logs.last().msg,
        )
    }
```

Reply:
You have a good point, there's exactly a test like this above (with/without default tag etc), first 7 or so tests cover the message cases, github doesn't show those lines by default 😄